ADDsynth.adoc

```diff
@@ -175,17 +181,39 @@ This is your oscillator and modulator-oscillator. Full documentation link:Oscill
 
 ==== modulation
 
-===== Amplitude
+===== Mix
+
+===== Ring
 image::imgs/add-synth-modulation-amplitude.png[]
 
-===== Frequency
+===== AM
+
+
+===== Phase
+The value of the modulator is applied as phase offset. That's what Yamaha calls FM.
+
+===== FM
+The value of the modulator is applied as frequency offset. Because of the wide applicable range it is nearly impossible to prevent aliasing at high amounts.
 image::imgs/add-synth-modulation-frequency.png[]
 
+===== PWM
+An enhancement of phase modulation with pairs om mirrored modulators
+
 ==== voice list
 image::imgs/add-synth-voice-list.png[]
 
+The AA button enables the anti aliasing sampling filter. Can be usfull when using high range frequency LFOs or FM.
+
 ==== resonance
 image::imgs/add-synth-resonance.png[]
 
+The resonance is a spectral modification of the oscillator that is applied "offline" while constructing the waveform.
+Attention: When the frequency is modulated the resonance spectrum is shifted, too.
```

Comment:
This attention block likely should use an admonition block https://docs.asciidoctor.org/asciidoc/latest/blocks/admonitions/